pages/about/dataaccess.vue

```diff
@@ -27,7 +27,7 @@
     <v-card class="woudc-note" tile max-width="68%" min-width="0px">
       <v-card-title>{{ $t('about.access.note1.title') }}</v-card-title>
       <v-card-text>
-        <i18n path="about.access.note1.body.template" tag="p">
+        <i18n path="about.access.note1.body.template" tag="span">
```

Comment:
I think the use of `tag="p"` was more acceptable here. Don't be afraid to use `<p>` to identify a group of text  (even if it's just 1 sentence) or identify as a paragraph. I'd say text in, for example, `<button>Cancel</button>` or `<li>List item</li>` don't need need to wrapped in a `<p>`.

Reply:
I was using `"span"` to avoid the margin-bottom from the `"p"`, but using `class="mb-0"` probably makes more sense for that.